pkgs/development/python-modules/gpib-ctypes/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+, setuptools
+, bump2version
+, wheel
+, watchdog
+}:
+
+buildPythonPackage rec {
+  version = "0.3.0";
+  pname = "gpib_ctypes";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-c9l6TNmM4PtbvopnnFi5R1dQ9o3MI39BHHHPSGqfjCY=";
+  };
+
+  nativeBuildInputs = [
```

Comment:
I was able to build this derivation without any of those nativeBuildInputs, are they necessary ?

Reply:
Have able to build also without it. Will update the derivation.